src/app/garfile/submit/success/index.njk

```diff
@@ -30,7 +30,7 @@
           GAR submitted
       </h1>
 
-      <p class="govuk-body-m">
+      <p class="govuk-body-m" style="color:white">
```

Comment:
is this inline style required?

Reply:
No, the inline is not required.
* I changed to match the gov design guide for the panel, the initial change was kinda of lazy e.g. just get it done with in the p tag.
* See result below.
<img width="1440" alt="Screenshot 2024-07-03 at 12 41 11" src="https://github.com/UKHomeOffice/egar-public-site-ui/assets/123387957/f68c91a5-360d-4f84-b9b2-10d969522e28">